cfme/tests/services/test_provision_stack.py

```diff
@@ -259,6 +252,7 @@ def _cleanup_vms():
                 wait_for(lambda: provider.mgmt.delete_stack(stack_data['stack_name']),
                          delay=10, num_sec=800, message="wait for stack delete")
             stack_data['vm_name'].delete_from_provider()
+            catalog_item.orch_template.delete()
```

Comment:
Gotcha, thanks for explaining.

In that case, you'll need to repeat this for each _cleanup_vms() in the module. `test_reconfigure_service` and `test_retire_stack` both use catalog_item fixture, which uses template fixture. I don't see a the template getting cleaned up in their finalizers?